File: read_file.js
const fs = require('fs');
const path = require('path');

const filePath = path.join(__dirname, 'src', 'components', 'Dashboard.js');

try {
  const content = fs.readFileSync(filePath, 'utf8');
  console.log(content.substring(0, 20000));
} catch (e) {
  console.error('Error:', e.message);
}

File: read_file.py
import os

file_path = r"c:\Users\USER\OneDrive\Desktop\gopal\src\components\Dashboard.js"

try:
    with open(file_path, 'r', encoding='utf-8') as f:
        content = f.read()
        print(content[:15000])  # Print first 15000 chars
except Exception as e:
    print(f"Error: {e}")
